### DISCUSSION TITLE
How to get correlation value?

### DISCUSSION THREAD

**Discussion by Ovital** [community/community#54145] [Programming Help] [answered]
### Select Topic Area

Question

### Body

I want to get correlation value of two array. How to get?

**Comment by smart9205** [comment, marked as answer]
You can calculate correlation value easily using numpy.
Here is the code :
import numpy as np
x = np.array([1, 2, 3, 4, 5])
y = np.array([5, 4, 3, 2, 1])
corr_matrix = np.corrcoef(x, y)
corr_coeff = corr_matrix[0, 1]
